Jenkinsfile

```
@@ -14,7 +14,7 @@ pipeline {
         stage ("Test") {
             steps {
                 git 'https://github.com/eclipse/kuksa.integration.git'
-                sh """head -c 10000 /dev/urandom > testing/src/test/resources/dummy && \
+                sh """head -c 10000 /dev/urandom > testing/src/test/resources/randomBytes && \
```

I'm sorry for being unclear about this: Please remove the command that creates the file from the Jenkinsfile. While it is unproblematic right now, it can hide errors in the future: If e.g. the randomBytes file is removed from the Git repository, Jenkins would re-create the file and show green tests, while tests run locally on developer PCs would fail.